jordy1611/NH-JS-Rancid-Tomatillos
# Feature/comment section

### What’s this PR do?
- adds all comment functionality; user can add comments when logged in, and see comments that others have posted
- Adds testing coverage for comment functionality
- Refactors all tests except for App.test.js to account for Router changes, all tests are passing except for one in App.js
### Where should the reviewer start?
MovieInfo.js - this is where the info for the comments section is pulled and passed down from
### How should this be manually tested?
Click on a movie, you should see a comments section! Because the server holding the comments is being ran from a local, there should be no comments there. You can log in and add a comment though, and it will persist - you'll see the author of the comment as well as the contents of the comment.
### What are the relevant tickets?
Resolves #48, resolves #53
